xamarin/Xamarin.Forms
# Fix 8784 [UWP] Avoid Placeholder text overlapping Search Icon in SearchBar

### Description of Change ###

In UWP, changes the SearchBar placeholder to only have a ColumnSpan of 1, and changes it to a ContentPresenter with a ContentTemplate that shows an ellipsis in case the text is too long.

This avoids the overlap with the SearchIcon that became apparent with #8773.

### Issues Resolved ### 

- fixes #8784

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

If an app previously had a SearchBar with a Placeholder text long enough to overlap the SearchIcon, it will no longer do so, but end in an ellipsis.

### Before/After Screenshots ### 

Before these changes:
![fix-8784-before](https://user-images.githubusercontent.com/3807144/72147658-565d6780-339f-11ea-9213-66e843731f37.png)

After these changes:
![fix-8784-after](https://user-images.githubusercontent.com/3807144/72147888-d4217300-339f-11ea-8367-14e83920fc0b.png)


### Testing Procedure ###

On UWP, go to test case 8784. Reduce the width of the Window, and check that the Placeholder does not overlap the SearchIcon and ends in an ellipsis. 

### PR Checklist ###
